src/useCreateComponent.tsx

```diff
@@ -13,6 +14,7 @@ export const useCreateComponent = (
 
   React.useLayoutEffect(() => {
     if (wrapperDivRef.current !== null && component === null) {
+      (connectInstance as any).setReactSdkAnalytics(version);
```

Comment:
We can declare an interface above if to avoid this `as any` i.e. `as IConnectJSWithPrivateMethods`. Looking at this also makes me a bit nervous, would you mind doing this in a try catch and simply logging a warning in the try catch? Trying to avoid a possible future incident.